src/Metadata/Converter.py

```diff
@@ -79,3 +79,25 @@ def save_metadata_to_file(self, vector_metadata, file_name, append=False):
 
                 output_file.close()
                     
+    def read_data(self, file_name):
+        file = open(file_name, 'r')
+        vectors = []
+        vector_line = []
+        labels = []
+        label_line = []
+        while True:
+            line = file.readline()
+            #print line
+            line_split = line.split()
+            if not line:
+                break
+            elif line.find('#') == -1:
+                if len(line_split) >= 1:
+                    label_line.append(line_split[0])
+                    vector_line.append(line_split[1:])
+            else:
+                vectors.append(vector_line)
+                labels.append(label_line)
+                label_line = []
+                vector_line = []    
+        return vectors, labels     
```

Comment:
1. return our metadata
2. the vector list returned here is a list of string, it cannot be used for future task